app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] This should be done with the `author ` attribute (foreign key in this table) to reference the `user `model
`belongs_to :author, class_name: 'User'`